test/connectivity.jl

```diff
@@ -256,6 +256,14 @@
     @test @inferred(!isgraphical([1, 1, 1]))
     @test @inferred(isgraphical([2, 2, 2]))
     @test @inferred(isgraphical(fill(3, 10)))
+
+    # Test directed simple graphicality
+    for i in 1:3
+      sdg = SimpleDiGraph(10, 90)
+      @test @inferred(isdigraphical(indegree(sdg), outdegree(sdg)))
+    end
```

Comment:
Since `SimpleDiGraph(10,90)` is always complete, I suggest to remove the loop for:
```suggestion
    sdg = SimpleDiGraph(10, 90)
    @test @inferred(isdigraphical(indegree(sdg), outdegree(sdg)))
```